dropwizard-jsonapi/src/main/java/io/rtr/jsonapi/JSONAPI.java

```diff
@@ -171,9 +171,13 @@ public ResourceObjectBuilder(ResponseData<D> data) {
 		
 		public ResourceObjectImpl<D> build() {
 			ResourceObjectImpl<D> doc = new ResourceObjectImpl<D>();
-			doc.setData(data);
 			doc.setType(getType(data.attributes));
-			
+			//if there are no attributes set, make the section null so it is hidden in the response
+			if(data.attributes.toString().equals("BaseModel{id='null', created=null, updated=null}")) {
```

Comment:
I take it back, can't do that exactly. We explicitly set the id to null so it is in the 'id' field, but not the attributes. I can check the created or updated though.